alfresco-docker-tika/src/main/resources/engine_config.json

```diff
@@ -12,14 +12,292 @@
+        {"sourceMediaType": "application/x-cpio",                                                          "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/x-cpio",                                                          "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/x-cpio",                                                          "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/java-archive",                                                    "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/java-archive",                                                    "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/java-archive",                                                    "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/java-archive",                                                    "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/x-tar",                                                           "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/x-tar",                                                           "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/x-tar",                                                           "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/x-tar",                                                           "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/zip",                                                             "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/zip",                                                             "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/zip",                                                             "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/zip",                                                             "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/x-netcdf",                                                        "targetMediaType": "text/html"},
+        {"sourceMediaType": "application/x-netcdf",                                                        "targetMediaType": "text/plain"},
+        {"sourceMediaType": "application/x-netcdf",                                                        "targetMediaType": "application/xhtml+xml"},
+        {"sourceMediaType": "application/x-netcdf",                                                        "targetMediaType": "text/xml"},
+
+        {"sourceMediaType": "application/x-cpio",                                                          "targetMediaType": "text/html"},
```

Comment:
Lines 15-18 duplicates?

Reply:
It looks like it, thanks for spotting this.